gulp/tasks/eslint.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+import config from '../config';
+import gulp   from 'gulp';
+import eslint from 'gulp-eslint';
+
+gulp.task('eslint', () => {
+  // ESLint ignores files with "node_modules" paths.
+  // So, it's best to have gulp ignore the directory as well.
+  // Also, Be sure to return the stream from the task;
+  // Otherwise, the task may end before the stream has finished.
+  return gulp.src([config.scripts.src, '!app/js/templates.js', '!node_modules/**'])
```

Comment:
You shouldn't need to explicitly ignore `node_modules` since you're only sourcing `config.scripts.src`